tools/castle-editor/code/framedesign.pas

```diff
@@ -2682,8 +2679,7 @@ procedure TDesignFrame.MenuItemViewportCamera2DViewInitialClick(
 begin
   V := SelectedViewport;
   V.Setup2D;
-  ModifiedOutsideObjectInspector;
-  RecordUndo('Camera Setup for 2D View and Projection');
+  ;ModifiedOutsideObjectInspector('Camera Setup for 2D View and Projection');
 end;
```

Comment:
Accidental `;` at the beginning of line :)

Reply:
Fixed in 8f02ccff9a05f7cafc9e5eb9f74cd137140148e2